packages/apps-config/src/api/spec/polymesh.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+import schema from '@polymathnetwork/polymesh-types';
+
+const definitions: OverrideBundleDefinition = {
+    types: [
+      {
+        // on all versions
+        minmax: [0, undefined],
+        types: schema.types
+      }
+    ],
+    rpc: schema.rpc
+}
```

Comment:
Just run `yarn lint` locally, it will fix the indentation CI is not happy about.

Reply:
Done @jacogr